recipes-kernel/linux/linux-hostmobility-vf_4.19.bb

```diff
@@ -33,11 +33,19 @@ MACHINE_PATCHES:mx4-c61 = " \
     file://defconfig \
 "
 
+MACHINE_PATCHES:mx4-c61-rio = " \
+    file://0001-add-device-tree-for-c61-v61.patch \
```

Comment:
Based on the machine ovrerride you might not need to add defconfig and device three patch.